app/domain/rotation/conjur_facade.rb

```diff
@@ -38,15 +44,24 @@ def current_values(variables)
     #
     def update_variables(new_values, &rotator_code)
       @db.transaction do
+
+        # lock_secrets_for_resource_ids(new_values.keys)
```

Comment:
I think I'd rather see this removed. Given that it's commented out, it's obviously not affecting the success of the cukes. Until we have agreement about whether we need to do locking, it doesn't seem like it belongs here.

Reply:
@apotterri ok i'll remove this and add it as an issue that we can discuss later.